### ISSUE TITLE
It is fine if getString function not using async await?

### ISSUE THREAD

**Issue by gopelkujo** [xaldarof/encrypted-shared-preferences#8]
Since the `getString` function type is `Future<String>,` it is fine if we call `getString` without `await`? Because i've seen in the example code at pub dev call getString function without `await`. (only setter that use `await`)

**Comment by xaldarof** [comment]
Hello, thank you for your contribution. Could you show me where you see getString with a Future return result?
![image](https://github.com/user-attachments/assets/62d55937-743a-45b7-b6cf-51d25fc00624)

**Comment by gopelkujo** [comment]
Hey! thanks for your response.

After i did a double check, my package is different with this package. Mine is [encrypted_shared_preferences](https://pub.dev/packages/encrypted_shared_preferences) the discontinued one. Since i continuing a project by someone else before, I made a mistake because I identified the package incorrectly.

Sorry for this misunderstood. Migrating to this package asap.

Thanks for your work, hope i can contribute more in this project✨
